server/src/test/java/org/opensearch/search/query/QueryProfilePhaseTests.java

```diff
@@ -336,12 +336,12 @@ public void testMinScoreDisablesCountOptimization() throws Exception {
         assertEquals(TotalHits.Relation.EQUAL_TO, context.queryResult().topDocs().topDocs.totalHits.relation);
         assertProfileData(context, "MatchAllDocsQuery", query -> {
             assertThat(query.getTimeBreakdown().keySet(), not(empty()));
-            assertThat(query.getTimeBreakdown().get("score"), greaterThanOrEqualTo(100L));
+            assertThat(query.getTimeBreakdown().get("score"), greaterThanOrEqualTo(1L));
```

Comment:
It's not clear to me how `1L` vs `100L` was determined. If we're just testing if this is a positive number should we just use `0L`?

Reply:
Hey @jed326, thanks for the review. This test uses `RandomIndexWriter` to add random number of documents to the search index, so we expect varying score time in nanos here. A `0L` score time is not anticipated in this test because we expect the score count to be equal to 1 below.